<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChemFlow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_logo_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ChemFlow</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/logo.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>641</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_LigFlow">
     <attribute name="title">
      <string>LigFlow</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_DockFlow">
     <attribute name="title">
      <string>DockFlow</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_docking_software">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>148</y>
        <width>221</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Software used for docking</string>
      </property>
      <item>
       <property name="text">
        <string>AutoDock Vina</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PLANTS</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_dockingKeep">
      <property name="geometry">
       <rect>
        <x>37</x>
        <y>176</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Keep</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_docking">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>98</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Docking protocol</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_docking_nposes">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>177</y>
        <width>70</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Number of docked poses to generate or keep</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_dockingKeep_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>177</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>pose(s) per ligand</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_docking_configure_protocol">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>317</x>
        <y>148</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Configure your docking protocol</string>
      </property>
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dockingSoftware">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>150</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Software</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_inputFiles">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>10</y>
        <width>75</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Input files</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_docking_lig">
      <property name="geometry">
       <rect>
        <x>538</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Ligands MOL2 file</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_lig">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>60</y>
        <width>65</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Ligands MOL2 file</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>Ligands</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_docking_rec">
      <property name="geometry">
       <rect>
        <x>538</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Receptor MOL2 file</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rec">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>30</y>
        <width>65</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Receptor MOL2 file</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>Receptor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>220</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Configure execution</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>122</y>
        <width>56</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Name for this specific protocol</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_docking_protocol">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>120</y>
        <width>316</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Name for this specific protocol</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>default</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_docking_rec">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>30</y>
        <width>441</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Receptor MOL2 file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_docking_lig">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>60</y>
        <width>441</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Ligands MOL2 file</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>-53</x>
        <y>250</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Job queue</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_docking_job_queue">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>246</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Batch system used to control your jobs:&lt;ul&gt;&lt;li&gt;None: run locally in parallel&lt;/li&gt;&lt;li&gt;PBS: Portable Batch System for UNIX clusters&lt;/li&gt;&lt;li&gt;SLURM: Slurm Workload Manager for HPC clusters&lt;/li&gt;&lt;/ul&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PBS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SLURM</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_docking_configure_job_queue">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>207</x>
        <y>246</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configure the number of cores used...etc.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_docking_run">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>86</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Run the docking experiment</string>
      </property>
      <property name="text">
       <string>Dock</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_docking_archive">
      <property name="geometry">
       <rect>
        <x>462</x>
        <y>340</y>
        <width>96</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Compress all files from the specified project, protocol, and receptor in a single tar.gz archive&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Archive</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_docking_postprocess">
      <property name="geometry">
       <rect>
        <x>266</x>
        <y>340</y>
        <width>141</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Process and standardize the outputs for the specified project, protocol, and receptor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Postprocess</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_overwrite">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Overwrite results corresponding to the same project, protocol name and receptor name. Will overwrite docking poses, affinities... everything !&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Usefull if you've made an error in a protocol you've just run and want to correct it without manually deleting files.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Overwrite pre-existing results</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_postdock_logo">
      <property name="geometry">
       <rect>
        <x>229</x>
        <y>340</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/process.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_archivedock_logo">
      <property name="geometry">
       <rect>
        <x>424</x>
        <y>340</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/archive.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_dock_logo">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>340</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/run.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>640</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ScoreFlow">
     <attribute name="title">
      <string>ScoreFlow</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_Tools">
     <attribute name="title">
      <string>Tools</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="actionLoad_project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGitHub"/>
    <addaction name="actionReport_issue"/>
    <addaction name="actionTutorial"/>
    <addaction name="separator"/>
    <addaction name="actionDebug_mode"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_project">
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionLoad_project">
   <property name="text">
    <string>Load project</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="actionReport_issue">
   <property name="text">
    <string>Report issue</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDebug_mode">
   <property name="text">
    <string>Debug mode</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
